doc/cephfs/client-auth.rst

```diff
@@ -146,3 +146,45 @@ The optional ``{network/prefix}`` is a standard network name and
 prefix length in CIDR notation (e.g., ``10.3.0.0/16``).  If present,
 the use of this capability is restricted to clients connecting from
 this network.
+
+Filesystem information restriction
+==================================
+
+If desired, the monitor cluster can present a limited view of the filesystems
+available. In this case, the monitor cluster will only inform clients about
+filesystems specified by the administrator. Other filesystems will not be
+reported and commands affecting them will be ignored as if the filesystems
+did not exist.
+
+::
+
+ client.0
+     key: AQAz7EVWygILFRAAdIcuJ12opU/JKyfFmxhuaw==
+     caps: [mon] allow r, allow fsid=1, allow fsid=2
+
+In this case, the monitor cluster will display information about the MDS
+daemons associated with file system IDs 1 and 2. Standby MDS daemons will
+always be displayed. Note that information about restricted MDS daemons and
+filesystems may become available by other means, such as ``ceph health detail``.
+
+MDS communication restriction
+=============================
+
+By default, user applications may communicate with any MDS, whether or not
+they are allowed to modify data on an associated filesystem (see
+``Path restriction`` above). To restrict communication to MDS daemons
+associated with a particular filesystem (or filesystems), use:
+
+::
+
+ client.0
+     key: AQAz7EVWygILFRAAdIcuJ12opU/JKyfFmxhuaw==
+     caps: [mds] allow rw, allow fsid=1
+     caps: [mon] allow r, allow fsid=1
+     caps: [osd] allow rw tag cephfs data=cephfs_a
```

Comment:
I guess we also need to extend these OSDCaps to take an FSCID. Not urgent though.

Reply:
Well, the OSD does not know the fscid. We could tag the pools with it, though.

Reply:
The pools tags are good enough now I think. I don't believe we want to move forward anymore with shared data pools. The feature is no longer attractive with the pg_autoscaler.